src/style-spec/reference/v8.json

```diff
@@ -3816,7 +3816,10 @@
       ],
       "sdk-support": {
         "basic functionality": {
-          "js": "2.3.0"
+          "js": "2.3.0",
+          "android": "10.6.0",
+          "ios": "10.6.0",
+          "macos": "10.6.0"
```

Comment:
I'm not sure macos is supported tbh...

Reply:
@karimnaaji do you know the use-case of macOS in these entries? I see it's marked everywhere where we added support for Android and iOS, but I assume it might not be needed anymore. Should we still include it?

Reply:
I think macOS is for the macOS SDK: https://mapbox.github.io/mapbox-gl-native/macos/0.16.0/. But if we don't publish new versions anymore as part of iOS deployment, then we should remove it.